tasks/rh-sign-image/tests/test-rh-sign-image-timeout.yaml

```diff
@@ -0,0 +1,73 @@
+---
+apiVersion: tekton.dev/v1
+kind: Pipeline
+metadata:
+  name: test-rh-sign-image-timeout
+  annotations:
+    test/assert-task-failure: "run-task"
+spec:
+  description: Test a scenario when the IR processing times out
```

Comment:
I'm probably missing something obvious, but what is making this test case time out?

Reply:
It's because the `requestTimeout` parameter of the task is set to `1`. And this is passed to the `wait-for-ir` call. So the mock would mark the IR as passed in 10 seconds, but it will time out before that happens.

Reply:
I see. It makes sense now. It may be worth writing pretty much exactly that in the description, but not required. I'll leave it up to you

Reply:
ok, added.

Reply:
oops, thanks. turns out I accidentally pushed it to the main repo and not my fork. updated now.

Reply:
looks good, thanks